src/resources/postcss/tribe-events-full.pcss

```diff
@@ -14,6 +14,10 @@
 
 :root {
 	--featured-event-bg: #0ea0d7;
+	--twentynineteen_font__body: "NonBreakingSpaceOverride", "Hoefler Text", "Baskerville Old Face", Garamond, "Times New Roman", serif;
```

Comment:
I don't know what the convention for CSS variable names are, but I would stick to one here. The one above uses dashes, but you're using underscores.